gax-grpc/src/main/java/com/google/longrunning/OperationsSettings.java

```diff
@@ -32,98 +32,52 @@
 
 // AUTO-GENERATED DOCUMENTATION AND CLASS
 /** Settings class to configure an instance of {@link OperationsClient}. */
 @Generated("by GAPIC v0.0.5")
 @BetaApi
 public class OperationsSettings extends ClientSettings<OperationsSettings> {
-  private final UnaryCallSettings<GetOperationRequest, Operation> getOperationSettings;
-  private final PagedCallSettings<
-          ListOperationsRequest, ListOperationsResponse, ListOperationsPagedResponse>
-      listOperationsSettings;
-  private final UnaryCallSettings<CancelOperationRequest, Empty> cancelOperationSettings;
-  private final UnaryCallSettings<DeleteOperationRequest, Empty> deleteOperationSettings;
+
+  public static final OperationsSettings create(OperationsStubSettings stub) throws IOException {
+    return new OperationsSettings.Builder(stub.toBuilder()).build();
+  }
 
   /** Returns the object with the settings used for calls to getOperation. */
   public UnaryCallSettings<GetOperationRequest, Operation> getOperationSettings() {
-    return getOperationSettings;
+    return ((OperationsStubSettings) getStubSettings()).getOperationSettings();
   }
 
   /** Returns the object with the settings used for calls to listOperations. */
   public PagedCallSettings<
           ListOperationsRequest, ListOperationsResponse, ListOperationsPagedResponse>
       listOperationsSettings() {
-    return listOperationsSettings;
+    return ((OperationsStubSettings) getStubSettings()).listOperationsSettings();
   }
 
   /** Returns the object with the settings used for calls to cancelOperation. */
   public UnaryCallSettings<CancelOperationRequest, Empty> cancelOperationSettings() {
-    return cancelOperationSettings;
+    return ((OperationsStubSettings) getStubSettings()).cancelOperationSettings();
   }
 
   /** Returns the object with the settings used for calls to deleteOperation. */
   public UnaryCallSettings<DeleteOperationRequest, Empty> deleteOperationSettings() {
-    return deleteOperationSettings;
+    return ((OperationsStubSettings) getStubSettings()).deleteOperationSettings();
   }
 
   @BetaApi("A restructuring of stub classes is planned, so this may break in the future")
   public OperationsStub createStub() throws IOException {
-    if (getTransportChannelProvider()
-        .getTransportName()
-        .equals(GrpcTransportChannel.getGrpcTransportName())) {
-      return GrpcOperationsStub.create(this);
-    } else {
-      throw new UnsupportedOperationException(
-          "Transport not supported: " + getTransportChannelProvider().getTransportName());
-    }
-  }
-
-  /** Returns a builder for the default ExecutorProvider for this service. */
-  public static InstantiatingExecutorProvider.Builder defaultExecutorProviderBuilder() {
-    return InstantiatingExecutorProvider.newBuilder();
-  }
-
-  /** Returns a builder for the default credentials for this service. */
-  public static GoogleCredentialsProvider.Builder defaultCredentialsProviderBuilder() {
```

Comment:
We can't remove this (or the prior method) because it's part of the stable generated surface.

Reply:
Sure thing; restored those methods.